scss/_patterns_notifications.scss

```diff
@@ -119,6 +119,7 @@ $notification-text-margin-bottom: $spv--large - $spv-nudge;
       .p-notification__title,
       .p-notification__message {
         display: inline;
+        word-wrap: break-word;
```

Comment:
This one is likely not needed.
This is specifically for inline version, but it will inherit any styling from the default one. So if it's specified for general `p-notification__message` (as below), it will also work here.